README.md

```diff
@@ -0,0 +1,60 @@
+![HD2 Community API](https://i.imgur.com/I1wosdV.png)
+
+A proxy API providing endpoints to access Helldivers 2 API data, both RAW API Data and Formatted API Data.
+<br />
+
+<details>
+<summary>Project Setup</summary>
+<details>
+<summary>Environment Setup</summary>
+
+For a sanitary environment, dev work should be done inside a [Virtual Environment](https://docs.python.org/3/library/venv.html)<br>
+
+
+<pre><code>python -m pip install --user --upgrade pip
```

Comment:
normally in markdown you would write this with three backticks followed by python:
'''python (but ` instead of ')

Reply:
Change implemented in incoming commit